docs/installation/instructions.rst

```diff
@@ -43,20 +43,19 @@ PyPI Installation
 
    $ sudo apt-get install python3-venv
 
-3. Install Pulp::
+3. Install Pulp (install ``pulpcore-plugin``, and ``pulpcore`` itself will be installed as
+a dependency)::
```

Comment:
Maybe we should suggest that the user install the plugin they want, and pulpcore-plugin will be picked up as a dep. Is there any reason a user would install pulp without a plugin?

Reply:
That makes sense to me. Something like:

```rst
3. Installing a plugin will automatically install ``pulpcore`` and ``pulpcore-plugin``::

    pip install pulp_file
```